gtfs/spec/en/reference.md

```diff
@@ -379,20 +379,25 @@ Shapes describe the physical path that a vehicle takes, and are defined in the f
 
 File: **Optional**
 
-This table is intended to represent schedules that don't have a fixed list of stop times. When trips are defined in frequencies.txt, the trip planner ignores the absolute values of the **arrival_time** and **departure_time** fields for those trips in [stop_times.txt](#stop_timestxt). Instead, the **stop_times** table defines the sequence of stops and the time difference between each stop.
+This table is used to represent two types of transit service:
+
+* frequency-based trips - Service that does not have a schedule of stop times (**exact_times** is **0** or **empty**).  Instead, operators make their best effort to serve stops periodically based on a headway.  For example, a rider can expect a bus to depart approximately every 15 minutes, but the vehicle will not attempt to adhere to a preset scheduled departure time such as 10:00 am.  Therefore, in the absence of real-time information, every subsequent departure is expected with a interval of **headway_secs** from the most recent departure. 
+* headway-defined schedules - Service that adheres to a schedule (**exact_times** is **1**).  This table can be used to represent simple scheduled service based on a fixed interval without fully listing out all scheduled trips via the [trips.txt](#tripstxt) and [stop_times.txt](#stop_timestxt) files.  Instead, the scheduled start time is defined in this table along with the set time between each departure (i.e., headway).  Consumers will then expand this to represent scheduled service as if each of these trips were defined in [trips.txt](#tripstxt) and [stop_times.txt](#stop_timestxt). 
+
```

Comment:
I am having trouble with the term "headway-defined schedules" for service that adheres to a schedule (**exact_times** is **1**). I keep thinking about "headway" as particularly meaningful because of its context with customers. i.e.. _"Service every 10 minutes"_ shows how headway is presents a "schedule" in customer-facing context. (exact_times = 0).

In transit operations, the similar term "headway-based" seems to often indicate the absence of an exact schedule. Or, at least appeared to, based on a Google search:

* [Seattle Transit blog](https://www.seattletransitblog.com/2012/11/15/transit-information-systems-schedules/): "Headway based schedules should primarily be used when time savings for riders from improved reliability due to active headway maintenance exceeds the added uncertainty or wait time due to lack of a set schedule."
* TRB Paper, [_Reliability Improvement in Short Headway Transit Services Schedule- and Headway-Based Holding Strategies_](http://trrjournalonline.trb.org/doi/abs/10.3141/2143-09): "Holding vehicles can improve reliability, resulting in shorter travel times and less crowding. In this paper both schedule-based and headway-based holding strategies in short headway services are analyzed."